### PR TITLE
ftp: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by pxgamer** [comment]
Sure 👍

File: pages/windows/ftp.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# ftp
+
+> Interactively transfer files between a local and remote FTP server.
```
Comment (sbrl) [inline pages/windows/ftp.md]: Windows machines don't have this enabled by default IIRC, so perhaps a note saying `Must be enabled in the Windows features settings` or something might be appropriate?

File: pages/windows/ftp.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+`ftp -n {{host}}`
+
+- Run a file containing a list of FTP commands:
+
+`ftp -s:{{path/to/file}} {{host}}`
+
+- Download multiple files (glob expression):
+
+`mget {{*.png}}`
```
Comment (sbrl) [inline pages/windows/ftp.md]: Is `mget` part oft he same command?

File: pages/windows/ftp.md (hunk 1 above)
Comment (pxgamer) [reply pages/windows/ftp.md]: They don't? 🤔 I never enabled it on my PC.

File: pages/windows/ftp.md (hunk 2 above)
Comment (pxgamer) [reply pages/windows/ftp.md]: `mget`, `mput` and `mdelete` are used once the interactive shell has been started.

Comment (sbrl) [reply pages/windows/ftp.md]: Ahhh I see.

**Pull request merged**